Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -680,11 +716,11 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
       }
       if (src == GIndex::Source::TPC && mFillTracksTPC) {
```

Comment:
and then, instead of having separate block (with mostly repeating code) for every type of barrel track, you can have a single block
````
auto contributorsGID = recoData.getSingleDetectorRefs(src);
if (contributorsGID[GIndex::Source::ITS].isIndexSet()) {
 // process ITS part from contributorsGID[GIndex::ITS]
}
if (contributorsGID[GIndex::Source::TPC].isIndexSet()) {
 // process TPC part from contributorsGID[GIndex::TPC]
}
// etc.

````